activestorage/lib/active_storage/engine.rb

```diff
@@ -117,6 +117,20 @@ class Engine < Rails::Engine # :nodoc:
       end
     end
 
+    initializer "active_storage.queues" do
+      config.after_initialize do |app|
```

Comment:
do we need an after_initialize here?

Reply:
I think we do, since `config.active_storage.queue` may be set in an initializer.

Reply:
oh, that is true. For other frameworks we use on_load but I think we don't/can't have a hook for Active Storage.